websites/J/JKAnime/presence.ts

```diff
@@ -0,0 +1,75 @@
+const presence = new Presence({
+    clientId: "731164832426098748"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",

```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/PreMiD/Presences/pullRequest?prid=6113512)